herbieapp/models/message_models_serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+from rest_framework.fields import JSONField
+
+
+class EntityUpdateMessage:
```

Comment:
These classes are no Models, so it's a bit confusing to put the file into the models directory.

Reply:
Yes i can understand that thought, i just believe that in the model package we can also have some data related classes that are not represent a table in the database. What is your opinion?